src/components/dialogs/generator-modification-dialog.js

```diff
@@ -271,11 +277,17 @@ const GeneratorModificationDialog = ({
         previousValues: generatorInfos?.reactiveCapabilityCurvePoints,
     });
 
+    const minimumReactivePowerRef = useRef();
```

Comment:
Can you add a comment here saying something like 
`// We need this here because we can't access lexical declaration 'minimumReactivePower' before initialization`
?

Reply:
Done